spec/grammar.dd

```diff
@@ -1616,6 +1616,7 @@ $(GNAME DeclarationBlock):
 $(GRAMMAR
 $(GNAME LinkageAttribute):
     $(D extern) $(D $(LPAREN)) $(GLINK LinkageType) $(D $(RPAREN))
+    $(D extern) $(D $(LPAREN)) $(GLINK MangleType) $(D $(RPAREN))
```

Comment:
That gives `extern ( MangleType )`. Should it be `extern ( C++, MangleType )` ?
Also it should appear after `extern ( C++, IdentifierLiest )`.